app/src/main/java/ch/epfl/sdp/map/MapsActivity.java

```diff
@@ -141,7 +142,7 @@ public void onRequestPermissionsResult(int requestCode, String[] permissions, in
     public void onMapReady(GoogleMap googleMap) {
         Game.getInstance().setMapApi(new GoogleMapApi(googleMap));
         Game.getInstance().setRenderer(this);
-
+        Game.getInstance().addToDisplayList(new Market(new GeoPoint( 6.141384, 46.214278))); // for demo add Market in GVA

```

Comment:
If the goal of this pull request is tp have a market at some pre defined spot, you don't actually need to put it on master (they told we can have a separate branch for hardcoded values used for demoing)